src/alchemlyb/convergence/convergence.py

```diff
@@ -51,16 +52,13 @@ def forward_backward_convergence(df_list, estimator='mbar', num=10):
     logger.info('Start convergence analysis.')
     logger.info('Check data availability.')
 
-    if estimator.lower() == 'mbar':
```

Comment:
@orbeckst I have thought about this and the issue is with AutoMBAR. The AutoMBAR is not upper case so you cannot get it with `getattr(alchemlyb.estimators, estimator)`, the solution that I purposed is to use AutoMBAR to replace the MBAR. For pymbar >= 4, this would also allow us to deprecate AutoMBAR easily.